src/lib/builders/combobox/create.ts

```diff
@@ -94,6 +94,32 @@ export function createCombobox<
 		},
 	});
 
+	const trigger = makeElement(name('trigger'), {
+		stores: [listbox.elements.trigger],
+		returned: ([$trigger]) => {
+			return {
+				...omit($trigger, 'action', 'id'),
+			};
+		},
+		action: (node: HTMLElement): MeltActionReturn<ComboboxEvents['trigger']> => {
+			const unsubEvents = executeCallbacks(
+				addMeltEventListener(node, 'click', () => {
+					const inputEl = document.getElementById(get(listbox.elements.trigger).id);
+					if (inputEl) {
+						inputEl.focus();
```

Comment:
You should probably only focus when it's being opened, I'd gather 🤔

Reply:
✅